pkg/ui/workspaces/cluster-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cockroachlabs/cluster-ui",
-  "version": "22.1.0-prerelease-3",
+  "version": "22.1.0-prerelease-4",
```

Comment:
@maryliag just bumped this to `22.2.0-prerelease-1` in #82476. Do you want this new component to be part of the 22.1 series, or will it go out in the fall?